Main.java

```diff
@@ -3,4 +3,86 @@ public static void main(String[] args){
         System.out.println ("Test");
     }
 
+    //sleapIn
+    public boolean sleepIn(boolean weekday, boolean vacation) {
+        return (!weekday || vacation);
+    }
+
+    //monkey trouble
+    public boolean monkeyTrouble(boolean aSmile, boolean bSmile) {
```

Comment:
two alternatives:
1. use ?: operator (please get familiar with it)
2. XOR operator (^)